src/Javis.jl

```diff
@@ -309,6 +308,10 @@ function render(
     started_encoding_video = false
 
     filecounter = 1
+    binpath = joinpath(FFMPEG.FFMPEG_jll.PATH_list[end], "ffmpeg") #is ffmpeg bin always at the end ? 
```

Comment:
can't we use `ffmpeg` like here: https://docs.juliahub.com/FFMPEG/TGvga/0.4.0/autodocs/#FFMPEG.@ffmpeg_env-Tuple%7BAny%7D

Reply:
yes indeed ,